src/EditorFeatures/Core.Wpf/InlineErrors/InlineErrorAdornmentManager.cs

```diff
@@ -0,0 +1,216 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Windows.Controls;
+using Microsoft.CodeAnalysis.Editor.Implementation.Adornments;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Adornments;
+using Microsoft.VisualStudio.Text.Classification;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Formatting;
+using Microsoft.VisualStudio.Text.Tagging;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineErrors
+{
+    internal class InlineErrorAdornmentManager : AdornmentManager<InlineErrorTag>
+    {
+        private readonly IClassificationTypeRegistryService _classificationRegistryService;
+        private readonly IClassificationFormatMap _formatMap;
+        private readonly Dictionary<IMappingTagSpan<InlineErrorTag>, SnapshotPoint> _tagSpanToPointMap;
+
+        public InlineErrorAdornmentManager(IThreadingContext threadingContext,
+            IWpfTextView textView, IViewTagAggregatorFactoryService tagAggregatorFactoryService,
+            IAsynchronousOperationListener asyncListener, string adornmentLayerName,
+            IClassificationFormatMapService classificationFormatMapService,
+            IClassificationTypeRegistryService classificationTypeRegistryService)
```

Comment:
nit: my preferences would be:

1. wrap IThreadingContext, so that all of these are starting on the same line.
2. wrap it all :)

Reply:
fixed